lua/kubectl/views/clusterrolebindings/init.lua

```diff
@@ -0,0 +1,62 @@
+local ResourceBuilder = require("kubectl.resourcebuilder")
+local definition = require("kubectl.views.clusterrolebindings.definition")
+local state = require("kubectl.state")
+local tables = require("kubectl.utils.tables")
+
+local resource = "clusterrolebindings"
+
+---@class Module
+local M = {
+  definition = {
+    resource = resource,
+    display_name = string.upper(resource),
+    ft = "k8s_" .. resource,
+    gvk = { g = "rbac.authorization.k8s.io", v = "v1", k = "clusterrolebinding" },
```

Comment:
what if someone has a cluster with v1beta1? i think this should be handled dynamically

Reply:
This is how it looks like in main, so we are not loosing any functionality with this code, although, we should have a different discussion on this since there is multiple ways to handle this.
For the core api's, I think we are very safe with the v1 currently

<img width="550" alt="image" src="https://github.com/user-attachments/assets/84729702-60ce-487e-82aa-3f1d7f9fc4c6" />